libshvbroker/src/brokerapp.cpp

```diff
@@ -574,6 +574,15 @@ std::string BrokerApp::resolveMountPoint(const shv::chainpack::RpcValue::Map &de
 	if(mount_point.empty()) {
 		shvWarning() << "cannot find mount point for device id:" << device_id.toCpon();// << "connection id:" << connection_id;
 	}
+	else {
```

Comment:
I think that brokerapp:776 is better place to solve this. It seems that there is something like that currently, but it is commented out